libs/core/src/components/sage-tooltip/sage-tooltip.scss

```diff
@@ -0,0 +1,172 @@
+:host {
+  /**
+
+    * @prop --background: Background of the tooltip
+    * @prop --color: Text color of the tooltip
+    * @prop --arrow-size: width and height of the arrow
+    * @prop --arrow-offset: arrow offset position
+
+    * @prop --overlay-border-radius: overlay border-radius
+    * @prop --overlay-font-size: overlay font-size
+    * @prop --overlay-line-height: overlay line-height
+    * @prop --overlay-padding: overlay padding
+    *
+    * @prop --arrow-pointing-down: shape for downward facing arrow
+    * @prop --arrow-pointing-to-the-right: shape for right facing arrow
+    * @prop --arrow-pointing-to-the-left: shape for left facing arrow
+    * @prop --arrow-pointing-up: shape for upward facing arrow
+    */
+
+  --background: #000;
+  --color: #fff;
+  --width: 236px;
+  --arrow-size: 6px;
+  --arrow-offset: 14px;
+
+  --overlay-border-radius: 10px;
+  --overlay-font-size: 14px;
+  --overlay-line-height: 20px;
+  --overlay-padding: 8px 12px;
+
+  --arrow-pointing-down: var(--arrow-size) var(--arrow-size) 0;
+  --arrow-pointing-to-the-right: var(--arrow-size) 0 var(--arrow-size) var(--arrow-size);
+  --arrow-pointing-to-the-left: var(--arrow-size) var(--arrow-size) var(--arrow-size) 0;
+  --arrow-pointing-up: 0 var(--arrow-size) var(--arrow-size);
+
+  display: inline-block;
+  position: relative;
+
+  ::slotted([slot="content"]) {
+    white-space: normal;
+  }
+}
+
+:host(.sage-tooltip--has-html-content) {
+  width: var(--width);
+}
+
+.sage-tooltip__content {
+  background-color: var(--background);
+  border-radius: var(--overlay-border-radius);
+  color: var(--color);
+  font-size: var(--overlay-font-size);
+  line-height: var(--overlay-line-height);
+  opacity: 0;
+  padding: var(--overlay-padding);
+  position: absolute;
+  visibility: hidden;
+  width: max-content;
+  z-index: 1;
```

Comment:
Is the shadow missing here? Seems there is a med shadow in Figma. Maybe waiting on token?